Lib/pathlib.py

```diff
@@ -687,20 +695,23 @@ class Path(PurePath):
     """
     __slots__ = ()
 
-    def __new__(cls, *args, **kwargs):
+    def __init__(self, *args, **kwargs):
         if kwargs:
             msg = ("support for supplying keyword arguments to pathlib.PurePath "
                    "is deprecated and scheduled for removal in Python {remove}")
             warnings._deprecated("pathlib.PurePath(**kwargs)", msg, remove=(3, 14))
+        super().__init__(*args)
+
+    def __new__(cls, *args, **kwargs):
         if cls is Path:
             cls = WindowsPath if os.name == 'nt' else PosixPath
-        return cls._from_parts(args)
+        return super().__new__(cls, *args, **kwargs)
```

Comment:
@zooba is this the right way to call super in this context? Wondering if I should perhaps exclude `*args` and `**kwargs`?

Reply:
It might be safest to go directly to `object.__new__()` here, and then don't pass along any arguments. We ought to know exactly where we are in the inheritance hierarchy at this point (either the top, or one of our own classes that we know don't override `__new__`).